app/commands/git/sync.rb

```diff
@@ -0,0 +1,72 @@
+module Git
+  class Sync
+    include Mandate
+    initialize_with :track, :synced_to_git_sha
+
+    def call
+      sync! unless synced_to_head?
```

Comment:
I'm trying to work out the logic of this a bit. 

[Further down](https://github.com/exercism/v3-website/pull/291/files#diff-571fbde4167b7daec296052a69f079a05369faa3147dc34bdedce25eb6e60062R15) we do this: ```return concept.update!(synced_to_git_sha: head_git_concept.commit.oid) unless concept_needs_updating?```

But that will never get called if `synced_to_head?` here is true.

I can't quite work out if that's correct or not etc.

I'm wondering if we actually want this guard in this parent class, or if it should be down to each child class to do it' own guard.

What do you think?